Shopify/cli
# Notify timing fix

### WHY are these changes introduced?

We are notifying theme/theme extension updates when the files change but not when the workers have gone idle, which is the intention of the notify flag.



### WHAT is this pull request doing?

In this PR, we trigger the notifications when the workers have gone idle.

### How to test your changes?

1. Serve a theme or a theme extension with the notify flag enabled.
2. Change a liquid file and see the notification performed after the workers went idle. 

_E.g., File change at 19.57:59 and then notification at 19:58:00._

https://github.com/Shopify/cli/assets/5134470/d0ce8c26-d5a3-442a-a8b6-1c76583b2b6a

cc @karreiro

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
